### ISSUE TITLE
Can't close chats in mobile view.

### ISSUE THREAD

**Issue by richrd** [erming/shout#441]
The list of networks, channels and queries only shows the close button on hover. On a mobile/touch device this makes closing views almost impossible. No big deal ofcourse.

Anyway, could the close buttons be shown In the mobile view? I could make a PR for that :+1:

**Comment by astorije** [comment]
Hi @richrd,

> On a mobile/touch device this makes closing views almost impossible.

Actually, in mobile view, you have access to the `Leave`/`Close` button on the top right of the window. But I agree that in mobile mode, the `x` to close a channel should always be displayed.

> Anyway, could the close buttons be shown In the mobile view? I could make a PR for that

Did you send one, or are you still planning to send one? That would be great!

**Comment by richrd** [comment]
Didn't send one yet, and I actually forgot about this since I've rather busy with other things.
I'll try to get back to this when I have time. Thanks for reminding me :+1:

**Comment by astorije** [comment]
Hi @richrd, great! In the meantime, can you confirm that the buttons I am mentioning hold the functionality you are looking for?

**Comment by richrd** [comment]
I don't seem to have that button, was it implemented after version 0.51.1?

**Comment by astorije** [comment]
No it was there before.

Are you sure? I am talking about these guys on the right:

<img width="397" alt="IRC Query" src="https://cloud.githubusercontent.com/assets/113730/10077436/9c7777d6-62e0-11e5-852a-3d61e2ef4dec.png">
<img width="397" alt="IRC Channel" src="https://cloud.githubusercontent.com/assets/113730/10077437/9c79ef70-62e0-11e5-98cb-760b98a7f298.png">
<img width="397" alt="IRC Network" src="https://cloud.githubusercontent.com/assets/113730/10077438/9c7dc636-62e0-11e5-8ff9-dcb09d7eb78c.png">

**Comment by richrd** [comment]
Doh, they are there, but my theme hides them. I have my own theme which I think is based on Zenburn. The Zenburn theme has the following css:
```
/* Hide unnecessary buttons */
#windows .header .button,
#form #submit {
  display: none;
}
```
Since it's only CSS, and the functionality that I'm looking fore exists, I could probably just close this issue. Maybe submit a new one about the Zenburn theme. What do you think?

**Comment by astorije** [comment]
I think that by default (so, in the default CSS stylesheet), the closing button on the left menu should be visible by default on mobile version, since, as you pointed out, there is no hover state.

So I would leave the issue open and link to it in your PR that makes the closing button visible :-)

**Comment by richrd** [comment]
Great, I agree. Will also make my PR much simpler :)

**Comment by astorije** [comment]
@richrd be careful of the following: when a channel has unread messages, the number of messages is shown where the close button is. You might want to take that into consideration.

<img width="179" alt="screen shot 2015-09-26 at 03 42 21" src="https://cloud.githubusercontent.com/assets/113730/10115206/cf16fa74-6400-11e5-8a4d-cc1fd38905d5.png">
<img width="178" alt="screen shot 2015-09-26 at 03 42 29" src="https://cloud.githubusercontent.com/assets/113730/10115208/d3eb7a0c-6400-11e5-970a-3b2b03ca3bc5.png">


Let me know if you have any question or if you want to share implementation design details here.

**Comment by richrd** [comment]
Yep, I noticed that. I was thinking that the message count and the X button could just be placed side by side. Probably with the X on the right. At the moment shout already behaves a bit weird if the channel name is too long. Maybe that also needs a fix. I hope to get some progress done this week.

**Comment by astorije** [comment]
> Probably with the X on the right.

Sounds about right! Can't wait to see your work :-)
